vm-setup/roles/v1aX_integration_test/tasks/remediation.yml

```diff
@@ -127,11 +146,11 @@
     shell: |
        kubectl annotate bmh "{{ item }}" -n "{{ NAMESPACE }}" reboot.metal3.io/poweroff-
     with_items:
-      - "{{ MASTER_NODE_1 }}"
-      - "{{ MASTER_NODE_2 }}"
+      - "{{ MASTER_BMH_1 }}"
+      - "{{ MASTER_BMH_2 }}"
 
   - name: Wait until powered on master nodes become Ready
-    delegate_to: "{{ MASTER_0_IP.stdout }}"
+    delegate_to: "{{ MASTER_NODE_IP.stdout }}"
```

Comment:
that should not be delegated, it should just use the second kubeconfig

Reply:
True. But for now maybe we can keep it as it is now, because what you are suggesting will be done by https://github.com/metal3-io/metal3-dev-env/pull/385

Reply:
I ran the remediation locally and it was failed in line 152 because MASTER_NODE_IP cannot ssh path. As in v1alpha4 the node IP is 192.168.111.100 not 192.168.111.20. So if we keep this, v1alpha4 test will be failed. So it could be better to use kubeconfig and get the node status. It will work in both v1alpha3 and v1alpha4

Reply:
just use the VIP then it would solve all issues